### DISCUSSION TITLE
Windows 11 Driver Issues

### DISCUSSION THREAD

**Discussion by szammyboi** [NicoHood/HID#354] [Q&A]
Are their any known issues with this library and Windows 11? I had a joystick example working fine on a Windows 10 system. However, when I moved it to my personal computer running Windows 11, the operating system says that there are errors. The Arduino Zero shows up as a game controller under the devices menu but has a warning next to it that COM4 has a driver problem and under the device status it says "This device cannot start. (Code 10)." Any ideas?

**Comment by NicoHood** [comment]
I dont know yet of an issue, but Windows11 is quite new. Does the keyboard function work?

**Reply by szammyboi** [reply]
It seems to be an Arduino issue not this library itself. When I use the standard Arduino keyboard library the same issue is present.
